### DISCUSSION TITLE
Unable to Connect to Bing (Chunk error)

### DISCUSSION THREAD

**Discussion by CptAirwolf** [kubilus1/autoortho#222] [General]
Anybody else having issues with connecting to bing? I am unable too and keep getting infinite long loading times with error failed to get chunk data. This is quite frustrating. Is this on bings side or auto orthos side? wifi and connection is perfect.

**Comment by CptAirwolf** [comment]
I would really like some help. Got it to work for one flight earlier today and have been having this issue causing infinite loading times.

**Comment by CptAirwolf** [comment]
anybody? I have legit spent over 8 hours trying to get this to work.

**Comment by dpdurst** [comment]
What is your starting airport? I'm in Arizona and just tried and Bing maps downloaded fine with no issues or errors

**Comment by CptAirwolf** [comment]
I downloaded the Europe Section So I have attempted to start at Innsbruck (LOWI). Once out of every 10 tries it will randomly connect. But It has only happened twice.

**Comment by dpdurst** [comment]
Possibly then server issue on the one your connecting to. Not sure how AO is determining the server to use but they MS do have servers in EU and NA and probably other places that they stream from. For sure not happening here for me and I've been flying pretty much all weekend.
